lib/driver/pg.js

```diff
@@ -290,8 +290,21 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).nodeify(setType.bind(this));
```

Comment:
Well I don't mind if you commit your pull request, I don't want that things stop moving just because of a bit of rework. If you want to rework the whole function yourself, you would be welcome, but this is not necessary to proceed moving forward.